Added standard JS code linting and refactor code

kdhttps opened the pull request:
close #109 

1. Added Standard JS Code listing and formatting. Zero configurations, more details https://standardjs.com/ 
2. Added one more feature to check linting before commit in local machine more details here https://www.npmjs.com/package/husky

christian-hawk commented on test/app.init.test.js:
```diff
@@ -13,210 +13,199 @@ const file = chaiFiles.file
+    })
+
+    it('expect spTLSCert file to exist', () => {
+      // eslint-disable-next-line no-unused-expressions
```
Hey @kdhttps What if we use https://www.npmjs.com/package/eslint-plugin-chai-friendly instead of the comments?

kdhttps replied:
I tried but standard-js has no rules overwrite configuration facility. I guess we are trying to configuring this globally so we can configure it file_globally for specific file by adding `eslint-disable-next-line no-unused-expressions` flag at the top of the file. @christian-hawk what you think?

christian-hawk reviewed (changes requested):
Thanks for this @kdhttps ! 
Please check my minor change request feel free to discuss it.

kdhttps commented:
@christian-hawk so I've added [standardx](https://github.com/standard/standardx) by which we can use standard js feature as well as overwrite some custom rules so we have both features now, also fixed the `no-unused-expressions`. Please review

christian-hawk reviewed (approved):
Approved, nice job.
